src/nupic/regions/knn_classifier_region.py

```diff
@@ -32,19 +32,49 @@
 
 class KNNClassifierRegion(PyRegion):
   """
-  KNNClassifierRegion implements the k Nearest Neighbor classification algorithm.
-  By default it will implement vanilla 1-nearest neighbor using the L2 (Euclidean)
-  distance norm.  There are options for using different norms as well as
-  various ways of sparsifying the input.
-
-  Note: categories are ints >= 0.
+  KNNClassifierRegion implements the k Nearest Neighbor classification 
+  algorithm. By default it will implement vanilla 1-nearest neighbor using the 
+  L2 (Euclidean) distance norm.  There are options for using different norms as 
+  well as various ways of sparsifying the input.
+
+  .. note:: categories are ints >= 0.
+
+  :param maxCategoryCount: (int)
+  :param bestPrototypeIndexCount: (int) 
+  :param outputProbabilitiesByDist:  (bool)
+  :param k:  (int)
+  :param distanceNorm: (float) 
+  :param distanceMethod: (string)
+  :param distThreshold: (float)
+  :param doBinarization: (bool)
+  :param inputThresh: (float)
+  :param useSparseMemory: (bool)
+  :param sparseThreshold: (float)
+  :param relativeThreshold: (bool)
+  :param winnerCount: (int)
+  :param acceptanceProbability: (float) 
+  :param seed: (int)
+  :param doSphering: (bool)
+  :param SVDSampleCount: (int)
+  :param SVDDimCount: (int)
+  :param fractionOfMax: (int)
+  :param useAuxiliary: (int)
+  :param justUseAuxiliary: (int)
+  :param verbosity: (int)
+  :param replaceDuplicates: (bool) 
+  :param cellsPerCol: (int)
+  :param maxStoredPatterns: (int)
+  :param minSparsity: (float)
```

Comment:
I don't know any of this stuff. I'm fine leaving it like this if it is common knowledge.